microsoft-edge/webview2/get-started/winui2.md

```diff
@@ -196,6 +196,8 @@ Next, you install the **Microsoft.UI.Xaml** package for this project.  Microsoft
 
    For version 2.8.0 or later, in the **Dependencies** section at the bottom, **Microsoft.Web.WebView2** is listed.
 
+   Note that for HoloLens 2 development, the **Microsoft.Web.WebView2** package must be version 1.0.1722.45 or higher, which may be higher than the default.
```

Comment:
@JordanMandel / @mikehoffms can we also make a note here that developers have to be enrolled in WIP to use WebView2 on HoloLens 2?

Reply:
local draft:
For HoloLens 2 development, the **Microsoft.Web.WebView2** package must be version 1.0.1722.45 or higher, which may be higher than the default.  The WebView2 Preview is available in the Insider Preview for Microsoft HoloLens.  To access this preview, you must be enrolled in the Windows Insider Program; see [Start receiving Insider builds](/hololens/hololens-insider#start-receiving-insider-builds) in _Insider preview for Microsoft HoloLens_.  See also [Update HoloLens 2](/hololens/hololens-update-hololens).

https://learn.microsoft.com/en-us/hololens/hololens-insider#start-receiving-insider-builds
https://learn.microsoft.com/en-us/hololens/hololens-update-hololens

Reply:
done, see fresh https://github.com/MicrosoftDocs/edge-developer/pull/2567/files
you can close this Outdated comment; open fresh comment if needed